packages/cli/src/commands/versions/lint.ts

```diff
@@ -27,13 +27,21 @@ export const includedFilter = (name: string) =>
 
 // Packages that are not allowed to have any duplicates
 const FORBID_DUPLICATES = [
-  /^@backstage\/core$/,
-  /^@backstage\/core-api$/,
+  /^@backstage\/core-app-api$/,
   /^@backstage\/plugin-/,
 ];
 
+// There are some packages that ARE explicitly allowed to have duplicates since
+// they handle that appropriately. This takes precedence over FORBID_DUPLICATES
+// above.
+const ALLOW_DUPLICATES = [
+  /^@backstage\/core-plugin-api$/,
```

Comment:
This doesn't get matched in the above, but I guess it's nice to have it explicitly here anyway? 🤷

Reply:
yeah i know, felt it was clearer / more intentional and safer against any impl change we might make elsewhere :)